src/main-thread/nodes.ts

```diff
@@ -48,9 +48,9 @@ export function createNode(skeleton: TransferrableNode): RenderableElement {
 
   const namespace: string | undefined =
     skeleton[TransferrableKeys.namespaceURI] !== undefined ? getString(skeleton[TransferrableKeys.namespaceURI] as number) : undefined;
-  const node: HTMLElement | SVGElement = namespace
-    ? (document.createElementNS(namespace, getString(skeleton[TransferrableKeys.nodeName])) as SVGElement)
-    : document.createElement(getString(skeleton[TransferrableKeys.nodeName]));
+  const nodeName = getString(skeleton[TransferrableKeys.nodeName]);
+  let node: HTMLElement | SVGElement = namespace ? (document.createElementNS(namespace, nodeName) as SVGElement) : document.createElement(nodeName);
```

Comment:
`node` doesn't appear to be reassigned, can this be a `const`? I wonder why the linter didn't catch this.

```suggestion
  const node: HTMLElement | SVGElement = namespace ? (document.createElementNS(namespace, nodeName) as SVGElement) : document.createElement(nodeName);
```